README.md

```diff
@@ -242,4 +242,28 @@ Use the make command in the root directory of the project to run the app:
 make format-python
 ```
 
-This will run `isort`, `black` and `flake8`. Flake8 will ignore `E501 line too long`.
\ No newline at end of file
+This will run `isort`, `black` and `flake8`. Flake8 will ignore `E501 line too long`.
+
+## Testing
+### Setting up test environment variables
+
+```bash
```

Comment:
Worth mentioning what these env variables are for and providing a sensible example

Reply:
> Worth mentioning what these env variables are for and providing a sensible example

Fixed